data-structures/stacks-1.js

```diff
@@ -0,0 +1,16 @@
+export function getTop(stack) {
+  return stack.index === 0 ? undefined : stack.peek();
```

Comment:
This works, but `index` is supposed to be private and not accessible outside the Stack. You should use `stack.peek()` to get the value at the top (or undefined if there's nothing there).